pkgs/development/python-modules/eventlet/default.nix

```diff
@@ -50,8 +42,14 @@ buildPythonPackage rec {
     })
   ];
 
+  postPatch = ''
+    substituteInPlace tests/__init__.py \
+      --replace-fail "TEST_TIMEOUT = 2" "TEST_TIMEOUT = None"
```

Comment:
We should probably keep some timeout, but I think we can easily bump it to 20 seconds to be on the safe side for higher load machines.

Reply:
Why? The tests succeed without the timeout and Hydra has a default timeout for the entire build.
If we don't disable the timeout completely I'd rather leave it at upstream's default value.

Reply:
> Hydra has a default timeout for the entire build.

Relying on that is not that nice, since it will block the runner for a considerable amount of time.

Reply:
Alright, I removed the commit.